map/place_page_info.cpp

```diff
@@ -196,23 +210,17 @@ string Info::GetFormattedCoordinate(bool isDMS) const
                : measurement_utils::FormatLatLonAsDMS(ll.lat, ll.lon, 2);
 }
 
-string Info::GetRatingFormatted() const
+float Info::GetRatingRawValue() const
 {
-  if (!IsSponsored())
-    return string();
+  if (!IsSponsored() && !ShouldShowUGC())
+    return kIncorrectRating;
 
-  auto const r = GetMetadata().Get(feature::Metadata::FMD_RATING);
-  char const * rating = r.empty() ? kEmptyRatingSymbol : r.c_str();
-  int const size = snprintf(nullptr, 0, m_localizedRatingString.c_str(), rating);
-  if (size < 0)
-  {
-    LOG(LERROR, ("Incorrect size for string:", m_localizedRatingString, ", rating:", rating));
-    return string();
-  }
+  auto const rating = GetMetadata().Get(feature::Metadata::FMD_RATING);
```

Comment:
Wow, wow! We decided not to store rating in metadata, isn't it?

Reply:
Booking rating will be left in metadata. There is no ugc rating yet.

Reply:
Please, add at least comment that this is for sponsored objects only.

Reply:
Ok